feconf.py

```diff
@@ -608,13 +608,14 @@ def get_empty_ratings():
 ])
 
 # The header for the "Featured Activities" category in the library index page.
```

Comment:
The i18n id for the header of the "Featured Activities" category ...

Ditto below.